.github/workflows/code.yml

```diff
@@ -91,13 +99,19 @@ jobs:
         timeout-minutes: 10
 
       - name: Build binaries
-        run: cargo build --release --locked
+        run: cargo build --release --locked $BUILD_ARGS
+        env:
+          BUILD_ARGS: ${{ matrix.test.build-args }}
 
       - name: Run bash e2e tests
-        run: utils/e2e-tests/bash/run-tests "target/release/humanode-peer"
+        run: utils/e2e-tests/bash/run-tests $TEST_SUITE "target/release/humanode-peer" "target/release/wbuild/humanode-runtime/humanode_runtime.wasm"
```

Comment:
```suggestion
        run: utils/e2e-tests/bash/run-tests "$TEST_SUITE" "target/release/humanode-peer" "target/release/wbuild/humanode-runtime/humanode_runtime.wasm"
```

Reply:
Got it, fixed - https://github.com/humanode-network/humanode/pull/633/commits/089f56bc417410c7d0ea60cd5c4964f83d9019db )